src/dmd/backend/util2.c

```diff
@@ -38,6 +38,8 @@ void *ph_calloc(size_t nbytes);
 void ph_free(void *p);
 void *ph_realloc(void *p , size_t nbytes);
 
+extern "C" void internalFailureLogo(FILE* stream); // from dmd/mars.d
```

Comment:
"Logo" or "Log"?

Reply:
DMD already had a function called "logo" that printed its version. Hence "Logo" as this is an extended version of it, but I really don't care about the name ;-)

Reply:
Ok, I understand why it was called logo, but this isn't that.  Also, I suggested making methods begin with a verb, so my preference would be something like "printInternalFailure".